src/main/java/si/uni/prpo/group03/venueservice/VenueServiceApplication.java

```diff
@@ -3,10 +3,22 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import io.github.cdimascio.dotenv.Dotenv;
+
 @SpringBootApplication
 public class VenueServiceApplication {
 
 	public static void main(String[] args) {
+		// Load .env file
+        Dotenv dotenv = Dotenv.load();
+
+        // Set environment variables
+        System.setProperty("DB_HOST", dotenv.get("DB_HOST"));
+        System.setProperty("DB_PORT", dotenv.get("DB_PORT"));
+        System.setProperty("DB_NAME", dotenv.get("DB_NAME"));
+        System.setProperty("DB_USERNAME", dotenv.get("DB_USERNAME"));
+        System.setProperty("DB_PASSWORD", dotenv.get("DB_PASSWORD"));
+		
```

Comment:
Jaz bi za .env rajše uporabil konfiguracijo v application.properties, da nimava za brezveze dodanih knjižnic in extra kode.

.env file se v appliation.properties lahko importa tako:
`spring.config.import=optional:file:.env[.properties]`

Če boš spremenil, prosim še izbriši dotenv iz dependencyjev